components/feature/prompts/src/main/res/xml/feature_prompts_file_paths.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this file,
+   - You can obtain one at http://mozilla.org/MPL/2.0/.  -->
+<paths>
+    <cache-path name="feature-prompts-images" path="." />
```

Comment:
The equivalent XML file in feature-downloads (which I can't comment on) should use feature-downloads rather than feature-prompts-images

Reply:
Yes, I guess Github is just messing with me. The content of that file looked like feature-prompts.

Reply:
No worries :)